### PR TITLE
[WIP] Add image-manager service

### PR DESCRIPTION
**Pull request by jsiloto**
This may be better on a image-manager branch while the service itself is not functional, but I'll leave this here for comments

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -93,6 +93,20 @@ services:
         aliases:
           - devm
 
+  image-manager:
+    image: dojot/image-manager:latest
+    restart: always
+    depends_on:
```
Comment (mmagr) [inline docker-compose.yml]: mmm, this will depend on postgres and minio only, right?

File: docker-compose.yml [hunk 2]
```diff
@@ -93,6 +93,20 @@ services:
...
+      - mongodb
+      - postgres
+      - orion
+      - kafka
+    networks:
```
Comment (mmagr) [inline docker-compose.yml]: I think we might want to remove this (the aliasing of device-manager to devm is something that is not particularly necessary, and was kept only as some services would, eventually make calls to it using that old alias)

File: docker-compose.yml [hunk 3]
```diff
@@ -93,6 +93,20 @@ services:
         aliases:
           - devm
 
+  image-manager:
```
Comment (mmagr) [inline docker-compose.yml]: In retrospective, perhaps calling this service image-manager (mea culpa) is not a good idea (might be confused for actual images, like jpg files).

File: kong.config.sh [hunk 4]
```diff
@@ -63,6 +63,20 @@ authConfig "template"
...
 
+
+(curl -o /dev/null $kong/apis -s -S -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "image",
+    "uris": "/image",
+    "strip_uri": false,
+    "upstream_url": "http://imgm:5000"
```
Comment (mmagr) [inline kong.config.sh]: Here I think it'd be better if we use the service name itself, i.e. `http://image-manager:5000`

File: docker-compose.yml (hunk 3 above)
Comment (jsiloto) [reply docker-compose.yml]: Yeah, I also had that impression. Perhaps artifact-manager? or even firmware-manager...

File: docker-compose.yml (hunk 1 above)
Comment (jsiloto) [reply docker-compose.yml]: yes, but so far there is no minio

File: docker-compose.yml (hunk 3 above)
Comment (mmagr) [reply docker-compose.yml]: firmware-manager seems a good one.

File: docker-compose.yml (hunk 1 above)
Comment (mmagr) [reply docker-compose.yml]: done!

File: kong.config.sh [hunk 5]
```diff
@@ -70,7 +70,19 @@ PAYLOAD
...
 authConfig "device-manager"
 
-# auth service configuration
+
+(curl -o /dev/null ${kong}/apis -s -S -X POST \
+    --header "Content-Type: application/json" \
+    -d @- ) <<PAYLOAD
+{
+    "name": "image",
+    "uris": "/fw-image",
```
Comment (mmagr) [inline kong.config.sh]: using this as opposed to "/image" because that would break GUI (already uses that prefix)

**Pull request merged**